### PR TITLE
AA-1680 | Refactor

### PR DESCRIPTION
**Pull request by lchavarriaga**

### CODE REVIEW

**Comment by stephenfuqua** [comment]
@lchavarriaga when I try to run Admin App from this branch, after I have just reset all of the ODS/API databases using `initdev`, I get the following error on startup, and this probably explains why the E2E tests are failing.

```
https://localhost:44374/Setup/PostUpdateSetup

InvalidOperationException: Unable to resolve service for type 'EdFi.Ods.AdminApp.Management.ICompleteOdsFirstTimeSetupCommand' while attempting to activate 'EdFi.Ods.AdminApp.Web.Controllers.SetupController'.
```

File: Application/EdFi.Ods.AdminApp.Web/_Installers/OnPremInstaller.cs [hunk 1]
```diff
@@ -15,13 +14,7 @@ public class OnPremInstaller : CommonConfigurationInstaller
     {
         protected override void InstallHostingSpecificClasses(IServiceCollection services)
         {
-            services.AddTransient<IGetCloudOdsInstanceQuery, GetOnPremOdsInstanceQuery>();
-            services.AddTransient<IGetCloudOdsApiWebsiteSettingsQuery, GetOnPremOdsApiWebsiteSettingsQuery>();
-            services.AddTransient<IUpdateCloudOdsApiWebsiteSettingsCommand, UpdateOnPremOdsApiWebsiteSettingsCommand>();
-            services.AddTransient<IRestartAppServicesCommand, RestartOnPremAppServicesCommand>();
-            services.AddTransient<IFirstTimeSetupService, OnPremFirstTimeSetupService>();
             services.AddTransient<ITabDisplayService, OnPremTabDisplayService>();
-            services.AddTransient<ICompleteOdsFirstTimeSetupCommand, CompleteOnPremFirstTimeSetupCommand>();
```
Comment (stephenfuqua) [inline Application/EdFi.Ods.AdminApp.Web/_Installers/OnPremInstaller.cs:24]: `SetupController` still expects to receive a `ICompletedOdsFirstTimeSetupCommand` in its constructor, so this service setup is still required.

File: Application/EdFi.Ods.AdminApp.Web/_Installers/OnPremInstaller.cs [hunk 2]
```diff
@@ -5,7 +5,6 @@
 
 using Microsoft.Extensions.DependencyInjection;
 using EdFi.Ods.AdminApp.Management;
```
Comment (stephenfuqua) [inline Application/EdFi.Ods.AdminApp.Web/_Installers/OnPremInstaller.cs:7]: Please remove the unused `using` statement here.

**Comment by lchavarriaga** [comment]
New PR created. AA-1680 rework.

**Pull request closed without merging**